test/src/unit-constructor1.cpp

```diff
@@ -262,11 +263,12 @@ TEST_CASE("constructors")
 
         SECTION("std::tuple")
         {
-            const auto t = std::make_tuple(1.0, "string", 42, std::vector<int> {0, 1});
+            const auto t = std::make_tuple(1.0, std::string{"string"}, 42, std::vector<int> {0, 1});
```

Comment:
Because it was deduced as a `const char*` which you can not retrieve with `get`.